### DISCUSSION TITLE
Boy did I have fun with this talk ....

### DISCUSSION THREAD

**Discussion by jfmatth** [piku/piku#372] [Show and tell]
Enjoy

https://youtu.be/nt1bwaF7qxc?si=dnquSN23q9hSWJM3

**Comment by chr15m** [comment]
Great, will check it out, thanks for sharing!

**Comment by chr15m** [comment]
This is such a great talk John, love the energy!

Stuff I learned that I didn't know before:

- The alternate pronunciation. :exploding_head: I've been calling it "pee-ku" like you but I didn't realize some people call it "pie-ku". Now I'm curious how @rcarmo prounounces it. :grinning:
- Scaling. I've never actually used this feature (on node scaling to N connections is built in because of the way the server works), so it was cool to see how this works!

The question at the end got me thinking about running apps under separate users per-app for isolation/security. Wondering how hard it would be to set this up. :thinking:

Thanks again for sharing your talk. :+1:

**Reply by jfmatth** [reply]
Thank you, Chris, I do love working with the product.

Separate users would be great, then you could make a whole farm of PIKU servers :).  On a product like Proxmox, Piku runs great as a container, but would be amazing in a larger environment if you could run multiple users for isolation.

Thanks again.

**Reply by rcarmo** [reply]
actually, `uwsgi` sort of can do process isolation via `cgroups`. I just didn't want to mess with it at the time. And it can also use multiple UIDs, so we can try leveraging that.

**Comment by rcarmo** [comment]
Hey, this is great, thanks @jfmatth! I'm gonna add the video to the docs ASAP. 

Oh, and I pronounce it "pee-ku", because "I" is "ee" in Portuguese, but that never really mattered since associating it to the Raspberry Pi automatically made it different :)
